### ISSUE TITLE
Missing or stray quote in import file

### ISSUE THREAD

**Issue by kevinrobinson** [studentinsights/studentinsights#1777]
Missing or stray quote in line 236980
```
      /app/vendor/ruby-2.3.0/lib/ruby/2.3.0/csv.rb:1852:in `block (2 levels) in shift'
	/app/vendor/ruby-2.3.0/lib/ruby/2.3.0/csv.rb:1845:in `each'
	/app/vendor/ruby-2.3.0/lib/ruby/2.3.0/csv.rb:1845:in `block in shift'
	/app/vendor/ruby-2.3.0/lib/ruby/2.3.0/csv.rb:1805:in `loop'
	/app/vendor/ruby-2.3.0/lib/ruby/2.3.0/csv.rb:1805:in `shift'
	/app/vendor/ruby-2.3.0/lib/ruby/2.3.0/csv.rb:1747:in `each'
	/app/app/importers/file_importers/streaming_csv_transformer.rb:48:in `with_index'
	/app/app/importers/file_importers/streaming_csv_transformer.rb:48:in `each_with_index'
	/app/app/importers/file_importers/attendance_importer.rb:19:in `import'
	/app/app/importers/import_task.rb:136:in `block in import_all_the_data'
	/app/app/importers/import_task.rb:129:in `each'
	/app/app/importers/import_task.rb:129:in `import_all_the_data'
	/app/app/importers/import_task.rb:29:in `connect_transform_import'
	/app/app/jobs/import_job.rb:9:in `perform'
```

**Comment by kevinrobinson** [comment]
I can't see this looking at the file manually, although it might be that the file has since changed, I didn't look further.

**Comment by kevinrobinson** [comment]
Fixed in #1855
